src/components/item-display-panel/item-display-panel.module.ts

```diff
@@ -0,0 +1,9 @@
+import { ItemDisplayPanelComponent, ItemDisplayPanelContentDirective, ItemDisplayPanelFooterDirective } from './item-display-panel.component';
+import { NgModule, Input } from '@angular/core';
+import { ModalModule } from 'ngx-bootstrap/modal';
+
+@NgModule({
+    exports: [ItemDisplayPanelComponent, ItemDisplayPanelContentDirective, ItemDisplayPanelFooterDirective],
```

Comment:
Separate these out into a constant so we only need to have one array of the components/directives (See our other components as we do this there).